app/views/pageflow/editor/text_track_files/_text_track_file.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.(text_track_file, :configuration)
```

Comment:
Is this needed? The [configuration should be included by default](https://github.com/aviav/pageflow/blob/2525795da25baec58f70a6ffd25ff87a1920574d/app/views/pageflow/editor/files/_file.json.jbuilder#L1).